velox/docs/functions/spark/math.rst

```diff
@@ -96,6 +96,10 @@ Mathematical Functions
 
     Returns Euler's number raised to the power of ``x``.
 
+.. spark:function:: expm1(x) -> double
+
+    Returns Euler's number raised to the power of ``x`` minus 1 and this function is more accurate than exp(num) - 1whne the num is close to zero.
```

Comment:
> this function is more accurate than exp(num) - 1

This seems to be related to the implementation. Maybe move it near below implementation.

> // std::expm1 is more accurate than the expression std::exp(num) - 1.0 if num is close to zero.
> result = std::expm1(a);

nit: whne -> when

Reply:
Thanks and resolved.